diff --git a/dist/plugin.js b/dist/plugin.js
index 32e3adc2db38f7e770ca0ffc9bd2ac4fd3617ecb..8a0a9ffa958dd2f1900ebf7d49c50797fc312648 100644
--- a/dist/plugin.js
+++ b/dist/plugin.js
@@ -21,7 +21,8 @@ function pluginDocusaurus(context, pluginOptions) {
     const options = Object.assign(Object.assign({}, DEFAULT_PLUGIN_OPTIONS), pluginOptions);
     const inputFiles = options.inputFiles;
     const sidebar = options.sidebar;
-    const sidebarPath = path.resolve(siteDir, 'sidebars.js');
+    const sidebarFile = options.sidebarFile || 'sidebars.js';
+    const sidebarPath = path.resolve(siteDir, sidebarFile);
     const docsRoot = path.resolve(siteDir, options.docsRoot ? options.docsRoot : '');
     const outFolder = options.out !== undefined ? options.out : 'api';
     const out = docsRoot + (outFolder ? '/' + outFolder : '');
@@ -30,6 +31,7 @@ function pluginDocusaurus(context, pluginOptions) {
     delete options.inputFiles;
     delete options.out;
     delete options.docsRoot;
+    delete options.sidebarFile;
     return {
         name: 'docusaurus-plugin-typedoc',
         async loadContent() {
