diff --git a/dist/components/front-matter-component.js b/dist/components/front-matter-component.js
index 17584b7053b9e2d7ecd3789d857a51a25de37f37..b9f6b7da4736e48bc938803ba79890035d056f92 100644
--- a/dist/components/front-matter-component.js
+++ b/dist/components/front-matter-component.js
@@ -24,7 +24,7 @@ let DocsaurusFrontMatterComponent = class DocsaurusFrontMatterComponent extends
     }
     getSidebarLabel(page) {
         if (page.model.name === page.project.name) {
-            return page.url === page.project.url ? 'Globals' : 'README';
+            return page.url === page.project.url ? 'Summary' : 'README';
         }
         const item = page.navigation && page.navigation.children
             ? this.findNavigationItem(page.navigation.children, page.url)
