diff --git a/dist/components/front-matter-v3.component.js b/dist/components/front-matter-v3.component.js
index 1c3d6c73700e52e46efaa8028df647d847839e15..3623e56efd8f266f320260eeb7fb5854b11d28f3 100644
--- a/dist/components/front-matter-v3.component.js
+++ b/dist/components/front-matter-v3.component.js
@@ -37,7 +37,7 @@ ${Object.entries(yamlItems)
     getDefaultValues(page) {
         return {
             id: this.getId(page),
-            title: this.getTitle(page),
+            title: this.getTitle(page).replace(/&lt;/g, '<').replace(/&gt;/g, '>'),
         };
     }
     getId(page) {
diff --git a/dist/resources/helpers/reflection-title.js b/dist/resources/helpers/reflection-title.js
index 6d6ec1b8715c69ddbd9cd2c5f167ded5cd2b0d4e..5a30caa51b2d946056b44083b3b09ba6509e580d 100644
--- a/dist/resources/helpers/reflection-title.js
+++ b/dist/resources/helpers/reflection-title.js
@@ -9,7 +9,7 @@ function reflectionTitle() {
     title.push(this.model.name);
     if (this.model.typeParameters) {
         const typeParameters = this.model.typeParameters.map((typeParameter) => typeParameter.name).join(', ');
-        title.push(`‹**${typeParameters}**›`);
+        title.push(`&lt;${typeParameters}&gt;`);
     }
     return title.join(' ');
 }
diff --git a/dist/resources/helpers/signature-title.js b/dist/resources/helpers/signature-title.js
index d7062fd3224301edf42ceb1f4a31739919a38869..b9632f30b879adabd9edca359c6e0b3d2768dbcb 100644
--- a/dist/resources/helpers/signature-title.js
+++ b/dist/resources/helpers/signature-title.js
@@ -18,7 +18,7 @@ function signatureTitle(showSymbol) {
         md.push(`**${this.name}**`);
     }
     if (this.typeParameters) {
-        md.push(`‹${this.typeParameters.map((typeParameter) => `**${typeParameter.name}**`).join(', ')}›`);
+        md.push(`&lt;${this.typeParameters.map((typeParameter) => `**${typeParameter.name}**`).join(', ')}&gt;`);
     }
     const params = this.parameters
         ? this.parameters
diff --git a/dist/resources/helpers/type.js b/dist/resources/helpers/type.js
index 437ed84cf5d0565623387506cbb745d345998fef..6b8f7a7824978af9a02b6305ac3131a8f4955adc 100644
--- a/dist/resources/helpers/type.js
+++ b/dist/resources/helpers/type.js
@@ -28,7 +28,7 @@ function type() {
     if (this instanceof types_1.TypeOperatorType || this instanceof types_1.ReflectionType) {
         return this;
     }
-    return this ? this.toString().replace(/</g, '‹').replace(/>/g, '›') : '';
+    return this ? this.toString().replace(/</g, '&lt;').replace(/>/g, '&gt;') : '';
 }
 exports.type = type;
 function getReferenceType(model) {
@@ -36,7 +36,7 @@ function getReferenceType(model) {
         ? [`[${model.reflection.name}](${theme_1.default.handlebars.helpers.relativeURL(model.reflection.url)})`]
         : [model.name];
     if (model.typeArguments) {
-        reflection.push(`‹${model.typeArguments.map((typeArgument) => `${type.call(typeArgument)}`).join(', ')}›`);
+        reflection.push(`&lt;${model.typeArguments.map((typeArgument) => `${type.call(typeArgument)}`).join(', ')}&gt;`);
     }
     return reflection.join('');
 }
